doc/releases/releases.yml

```diff
@@ -17,6 +17,8 @@ releases:
     releases:
       - version: 19.2.0
         released: 2024-09-26
+      - version: 19.2.1
+        released: 2025-01-21
```

Comment:
We missed this.  Or does it need to align with the dates on packages?

Reply:
No, it needs to be the real release date.